### PR TITLE
Updated release notes for 0.4.0

### PR DESCRIPTION
**Pull request by OsiJr**
Resolves [AB#464](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/464)

### CODE REVIEW

File: builds/releasenotes.md [hunk 1]
```diff
@@ -7,11 +7,50 @@ Known Major Issues In Current Release
...
+  - Conversion Improvements
+    - Fixed issue with resetting some conversion types
+    - Increased support for additional types of FBX files
+    - Added drop down to set all spaces in a conversion rather than doing it individually
+    - Convert CMD takes a global offset now (`-globaloffset X Y Z`)
+    - Added Earth Centered, Earth Fixed support to conversion
+    - Fixed some issues with estimation of resolution for conversion
+    - Added support for additional SRID codes from customers and the community
+  - Improvements to SLPK
+    - Loading is ordered to focus on things closer to the camera
```
Comment (samsinsane) [inline builds/releasenotes.md]: This is kind of an awkward way of saying "Loading prioritizes objects closer to the camera".

File: builds/releasenotes.md [hunk 2]
```diff
@@ -7,11 +7,50 @@ Known Major Issues In Current Release
...
+    - Fixed some crahes that occurred when loading lots of SLPK files at the same time from fast disks
+    - Added ability to reset I3S models positions from the context menu
+  - Various rendering improvements
+    - The scene is scaled properly to avoid aliasing affects
+    - Improved performance when moving the mouse over the scene window with lots of polygon objects
+    - Added basic lighting to Polygon models (including existing Live Feed and SLPK files)
+  - macOS Fixes
+    - Improved consistency of visualisation to match OpenGL and D3D
```
Comment (samsinsane) [inline builds/releasenotes.md]: This doesn't mention Metal - I believe we still ship OpenGL for macOS?

File: builds/releasenotes.md [hunk 3]
```diff
@@ -7,11 +7,50 @@ Known Major Issues In Current Release
...
+    - Fixed a minor memory leak when resizing the window
```
Comment (samsinsane) [inline builds/releasenotes.md]: Does this apply to Metal, OpenGL or both?

File: builds/releasenotes.md [hunk 4]
```diff
@@ -7,11 +7,50 @@ Known Major Issues In Current Release
...
+  - Added basic geometric filters for UDS files
+  - Added support for relative paths when loading JSON projects
+  - Added new settings
+    - Now possible to display a colour gradient in conjunction with contours
+    - Option to always show the Euclideon logo on the screen
+    - License check out as part of login
+    - Invert X and Y for controller inputs
+  - Fixed lots of minor bugs:
+    - Fixed an issue where random transform were taking place when using the gizmo's in specific circumstances
+    - Horizontal lines generate more horizontally
```
Comment (samsinsane) [inline builds/releasenotes.md]: This doesn't really make sense.

File: builds/releasenotes.md [hunk 5]
```diff
@@ -7,11 +7,50 @@ Known Major Issues In Current Release
...
+    - Input events are no longer lost when focus is comes back to the application
```
Comment (samsinsane) [inline builds/releasenotes.md]: This should probably be:
> when focus comes back

File: builds/releasenotes.md [hunk 6]
```diff
@@ -7,11 +7,50 @@ Known Major Issues In Current Release
...
+    - Hotkeys no longer activate when the user is typing in a text field
+    - Increased the number of labels required to be on screen before graphical glitches occur (now millions rather than around a thousand)
+    - Going into and out of presentation mode many times quickly no longer stops the mouse working in the scene tab
+    - Media node images load correctly more often
+    - Sped up ultra long login and license checkout times that were occuring when in very slow network environments with machines that have very fast per core CPUs
```
Comment (samsinsane) [inline builds/releasenotes.md]: This doesn't sound like a Vault Client bug fix? Also, "ultra long" could probably be rephrased to "excessive".

File: builds/releasenotes.md (hunk 2 above)
Comment (OsiJr) [reply builds/releasenotes.md]: True- We've never shipped Metal before either.

**Comment by OsiJr** [comment]
@samsinsane, I'm fixing these issues

File: builds/releasenotes.md (hunk 4 above)
Comment (OsiJr) [reply builds/releasenotes.md]: Anyone that saw the bug would understand ;)

File: builds/releasenotes.md (hunk 6 above)
Comment (OsiJr) [reply builds/releasenotes.md]: It was technically a VDK bug but it was reported multiple times for Client so thought I should list it here as well as many Client users aren't aware there's a difference

File: builds/releasenotes.md (hunk 3 above)
Comment (OsiJr) [reply builds/releasenotes.md]: I"ve removed this section entirely as they are all fixes to a build that has never been released

**Pull request merged**